apps/prairielearn/src/tests/groupExam.test.js

```diff
@@ -0,0 +1,401 @@
+const { config } = require('../lib/config');
+const assert = require('chai').assert;
+const cheerio = require('cheerio');
+const fetch = require('node-fetch').default;
+const { step } = require('mocha-steps');
+
+const sqldb = require('@prairielearn/postgres');
+const sql = sqldb.loadSqlEquiv(__filename);
+
+const helperServer = require('./helperServer');
+const { idsEqual } = require('../lib/id');
+const { TEST_COURSE_PATH } = require('../lib/paths');
+
+let page, elemList;
+const locals = {};
+locals.helperClient = require('./helperClient');
+locals.siteUrl = 'http://localhost:' + config.serverPort;
+locals.baseUrl = locals.siteUrl + '/pl';
+locals.courseInstanceUrl = locals.baseUrl + '/course_instance/1';
+locals.assessmentsUrl = locals.courseInstanceUrl + '/assessments';
+
+const storedConfig = {};
+
+/**
+ * Switches `config` to new user, loads assessment page, and changes local CSRF token
+ * @param {Object} studentUser
+ * @param {string} assessmentUrl
+ * @param {String} authUin
+ * @param {Number} numCsrfTokens
+ */
+const switchUserAndLoadAssessment = async (studentUser, assessmentUrl, authUin, numCsrfTokens) => {
```

Comment:
Nit: we typically use the `function` keyword instead of arrow functions for top-level named functions like this.

```suggestion
async function (studentUser, assessmentUrl, authUin, numCsrfTokens) => {
```